gradle.properties

```diff
@@ -0,0 +1,20 @@
+# Project-wide Gradle settings.
+
+# IDE (e.g. Android Studio) users:
+# Gradle settings configured through the IDE *will override*
+# any settings specified in this file.
+
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+# Default value: -Xmx10248m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+
+android.disableAutomaticComponentCreation=true
```

Comment:
This will be a default behavior in AGP 8.0 so I already enforced it for this project. Surprisingly, it made e.g. sharing sources easier. More info: https://developer.android.com/studio/build/build-variants

Reply:
Thanks for adding a comment on that @wzieba !

PS: I am adding [this link](https://developer.android.com/studio/releases/gradle-plugin#disable-component-creation) as well as I wasn't sure when to look on the already share [link above](https://developer.android.com/studio/build/build-variants).